src/include/migraphx/shape_for_each.hpp

```diff
@@ -31,6 +31,10 @@
 namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
 
+/**
+ * Iterates the given function over the standard shape indices.
+ * Will iterate using standard strides if given a non-standard shape.
```

Comment:
This comment is very confusing. It should not say say anything about standard shape as that is irrelevent. This should just say: `Iterates the given function over the indices from the shape in order.`.